src/service.ts

```diff
@@ -366,22 +372,27 @@ export class GrpcService extends Service {
       // https://github.com/GoogleCloudPlatform/google-cloud-node/pull/1137#issuecomment-193315047
       return global['GCLOUD_SANDBOX_ENV'];
     }
-
+    if (config.grpc) {
+      this.grpc = config.grpc;
+      this.grpcVersion = config.grpcVersion || 'grpc/unknown';
```

Comment:
It would be nice if we had a way to track specifically which `grpc` module is getting used, and what version.  I think we're going to want that data.

Reply:
We cannot - I asked about it a few days ago and there is no way to tell from inside the code if we are using this or that version of gRPC. Asking clients to pass a version is probably the best solution here. Given that we control all clients, we can make sure we do pass the version all the time.

Alternatively, I can throw if `grpc` is passed and `grpcVersion` is not. WDYT?

Reply:
The only alternative I can think of here is having a common interface for grpc (core and js) that effectively returns the version and distribution.  

Would you mind filing a feature request against gRPC?  Should be additive.  I'd be ok with what's here for now, but I just want a plan to have better data as we go.